### PR TITLE
Add support to run prepare and verify scripts in normal JDBC test run

### PR DESCRIPTION
**Pull request by thephantomthief**
### Description

We have added support to test Major and Minor version upgrade on an
already populated database. Currently, we are using these prepare and
verify test scripts for testing version upgrade only.

This commit adds support in the JDBC test framework to run the prepare
and verify test scripts as part of standard JDBC test run (i.e. test run
not aimed for upgrade testing). We will run the prepare and verify
scripts in pairs together in case of standard JDBC test run.

Signed-off-by: Sharu Goel <goelshar@amazon.com>

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java [hunk 1]
```diff
@@ -61,6 +61,35 @@ public static void createTestFilesListUtil(String directory, String testToRun) {
         }
     }
 
+    // helper function to fetch the prefix of a prepare,
+    // verify or cleanup test file name
+    public static String getFileNamePrefix (String fileName) {
+        if (fileName.contains("-prepare")) {
+            return fileName.split("-prepare")[0];
```
Comment (surendravishnoi) [inline test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java]: this logic can break if some user happen to use these keyword in non upgrade test, may be we should use vu-prepare, vu-validate and so on

Comment (thephantomthief) [reply test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java]: Okay so should I rename the current upgrade test files too to have suffix "-vu-prepare", "-vu-validate" etc. ?

Comment (surendravishnoi) [reply test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java]: no, let us do that when we do the next part of upgrade support

**Review by rishabhtanwar29 — DISMISSED** [review]

File: test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java (hunk 1 above)
Comment (thephantomthief) [reply test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java]: Addressed comments in the latest commit

**Review by rishabhtanwar29 — APPROVED** [review]

**Review by surendravishnoi — APPROVED** [review]

**Review by kuntalghosh — APPROVED** [review]

**Pull request merged**